demisto_sdk/commands/common/hook_validations/incident_field.py

```diff
@@ -232,9 +232,7 @@ def is_valid_name(self):
             for word in name.split():
                 if word.lower() in bad_words:
                     error_message, error_code = Errors.invalid_incident_field_name(word)
-                    formatted_error = self.handle_error(error_message, error_code, file_path=self.file_path,
-                                                        should_print=False)
-                    click.secho(formatted_error, fg="yellow")
+                    click.secho(f"{self.file_path}: [{error_code}] - {error_message}".rstrip("\n") + "\n", fg="yellow")
```

Comment:
why? we should fail her

Reply:
This is IF100 which we said should not fail. but if it goes to handle_error it will add it to the report at the end - which it shouldn't. This way it will only print an error.